draft-ietf-dnsop-alt-tld.xml

```diff
@@ -126,11 +126,10 @@
       to signify that this is an alternative (non-DNS) namespace, and should
       not be looked up in a DNS context.</t>
       
-      <t>For technical clarity, the DNS on-the-wire format of a name that uses the
-      .alt label as a TLD always ends in the bytes 0x03616c7400.
-      Any list of bytes can precede those in the non-DNS name.
-      The use of ".alt" in this document uses the presentation format for the DNS,
-      but the presentation format for non-DNS protocols might be different.</t>
+      <t>The use of ".alt" for the pseudo-TLD in this document uses the presentation
```

Comment:
The use of ".alt" [...] uses the presentation format? :)

Reply:
(Sorry for breaking the thread.)

```
For clarity, this document uses the DNS presentation format for ".alt", corresponding ...
```

Reply:
great, thanks!

Reply:
Good catch; please see https://github.com/wkumari/draft-wkumari-dnsop-alt-tld/pull/19/commits/25ad13554cbd106bf98f66072bcd08fe240df169